docs/source/configuration/authorization.mdx

```diff
@@ -442,46 +441,38 @@ extend schema
 ```
 
-Using the `@policy` directive requires a [Supergraph plugin](../customizations/overview) to evaluate the authorization policies. You can do this with a [Rhai script](../customizations/rhai/), [coprocessor](../customizations/coprocessor), or [native plugin](../customizations/native). Refer to the following Rhai script and coprocessor examples for more information.
-
+Using the `@policy` directive requires a [Supergraph plugin](../customizations/overview) to evaluate the authorization policies. You can do this with a [Rhai script](../customizations/rhai/) or [coprocessor](../customizations/coprocessor). Refer to the following [Rhai script](#usage-with-a-rhai-script) and [coprocessor](#usage-with-a-coprocessor) examples for more information. (Although a [native plugin](../customizations/native) can also evaluate authorization policies, we don't recommend using it.)
 
 #### Combining policies with `AND`/`OR` logic
 
-A request must include _all_ elements in the inner-level `policies` array to resolve the associated field or type. In other words, the authorization validation uses **AND** logic between the elements in the inner-level `policies` array.
+Authorization validation uses **AND** logic between the elements in the inner-level `policies` array, where a request must include _all_ elements in the inner-level `policies` array to resolve the associated field or type. For the following example, a request would need `policy1` **AND** `policy2` **AND** `policy3` to be authorized:
 
 ```graphql
 @requiresScopes(policies: [["policy1", "policy2", "policy3"]])
 ```
 
-For the preceding example, a request would need `policy1` **AND** `policy2` **AND** `policy3` to be authorized.
-
-You can use nested arrays to introduce **OR** logic:
+Alternatively, to introduce **OR** logic you can use nested arrays. For the following example, a request would need `policy1` **OR** `policy2` **OR** `policy3` to be authorized:
 
 ```graphql
 @requiresScopes(policies: [["policy1"], ["policy2"], ["policy3"]])
 ```
 
-For the preceding example, a request would need `policy1` **OR** `policy2` **OR** `policy3` to be authorized.
-
-You can nest arrays and elements as needed to achieve your desired logic. For example:
+You can nest arrays and elements as needed to achieve your desired logic. For the following example, its syntax requires requests to have either (`policy1` **AND** `policy2`) **OR** just `policy3` to be authorized:
 
 ```graphql
 @policy(policies: [["policy", "policy2"], ["policy3"]])
 ```
 
-This syntax requires requests to have either (`policy1` **AND** `policy2`) **OR** just `policy3` to be authorized.
-
-
 ##### Usage with a Rhai script
 
-The `policies` argument contains a list of strings with no formatting constraints. That means you can use them to store Rhai code.
+The `policies` argument contains a list of strings with no formatting constraints, meaning you can use them to store Rhai code.
```

Comment:
I don't think we should recommend that folks do `eval` of arbitrary Rhai code from the subgraphs. There are a couple of problems:

1. You're suddenly giving developers who have access to subgraphs access to run code at the router level, which could let them get data they shouldn't have somehow? Maybe not the hugest issue, but worries me.
2. If someone writes invalid Rhai code in _any_ subgraph, they could break **every** operation which tests the policy—it wouldn't just cause that new field to get omitted, it'd cause entire requests to be aborted with an error.